crates/sui-json-rpc/src/read_api.rs

```diff
@@ -125,6 +126,51 @@ impl SuiRpcModule for ReadApi {
 
 #[async_trait]
 impl RpcFullNodeReadApiServer for FullNodeApi {
+    async fn get_normalized_move_module(
+        &self,
+        package: ObjectID,
+        module_name: String,
+    ) -> RpcResult<MoveNormalizedModule> {
+        let module = get_move_module(&self, package, module_name).await?;
+        Ok(MoveNormalizedModule::from(module))
```

Comment:
```suggestion
        Ok(module.into())
```

`.into()`is the more succinct and idiomatic way of doing conversion. Could you replace all usage of `::from` to `into()` in this PR?